### DISCUSSION TITLE
Need to add a repository from another account that i have colab with into a repository in my account with all commits and work

### DISCUSSION THREAD

**Discussion by cchristodoulou7** [community/community#36549] [New to GitHub]
i did a group assinment for uni
I need to create the same repository and add it from an excisting one from another acount that i have worked on (i am colaborator on it)! for future use 
when i choose to import repo from url it doent find it what do i do 
I need ALL the process and comits  is there a diffrent way this could work?

**Comment by DSDmark** [comment]
Hey @cchristodoulou7,

i m not sure but this could help you
check this out:- 
[**Transferring a repository**](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository)

**Reply by cchristodoulou7** [reply]
the other student that i coloborated with he also wants the repository is there a way to share ownership or something!!

**Comment by airtower-luna** [comment]
The simplest way to get the full history to an independent repository on GitHub is:

1. Clone the repository to your computer.
2. Create a new repository in your GitHub account.
3. Add the new repository on GitHub as a remote of your local repository.
4. Push to that remote.

See: [Git Basics - Working with Remotes](https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes)

However, if you still want to collaborate with the other student you might want to *fork* the repository on GitHub instead. That way you can later create PRs between the repositories, if you want. See: [Fork a repo](https://docs.github.com/en/get-started/quickstart/fork-a-repo)

If you want to actually share ownership of a single repository (assuming the other student wants that) you could create an organization on GitHub and transfer the repository to that organization.

**Reply by DSDmark** [reply]
That was the great answer @airtower-luna Wrote,
if you @cchristodoulou7 actually share ownership of a single repository, ya, you could create an organization.

If you go for that, creating organization, note that:-
According to that official **GitHub** documentation 

> It's not possible to convert an organization into a personal account, but you can create a new personal account and transfer the organization's repositories to it.
[**GitHub docs**](https://docs.github.com/en/organizations/managing-organization-settings/converting-an-organization-into-a-user)

In simpler word, read this first **GitHub docs** if you will create an organization.

And also this discussion (https://github.com/community/community/discussions/36406)

**Comment by ossanna16** [comment]
Hey there @cchristodoulou7 and thank you for asking a good question! I'm glad you found our community 🙂
